pkg/features/kube_features.go

```diff
@@ -664,6 +664,11 @@ const (
 	// Enable users to specify when a Pod is ready for scheduling.
 	PodSchedulingReadiness featuregate.Feature = "PodSchedulingReadiness"
 
+	// owner: @Huutomerkki, @peppi-lotta
+	//
+	// Enables pods starting in the order of their priorities
+	PodStartingOrderByPriority featuregate.Feature = "PodStartingOrderByPriority"
```

Comment:
it depends on how we frame this work. If it's bugfix, then we can default to enable. If it's a RFE, then we will need the KEP process.